env0/resource_template.go

```diff
@@ -401,11 +425,22 @@ func templateCreatePayloadFromParameters(prefix string, d *schema.ResourceData)
 // Reads template and writes to the resource data.
 func templateRead(prefix string, template client.Template, d *schema.ResourceData) error {
 	pathPrefix := "path"
+	terragruntTfBinaryPrefix := "terragrunt_tf_binary"
+
 	if prefix != "" {
-		pathPrefix = prefix + ".0.path"
+		pathPrefix = prefix + ".0." + pathPrefix
+		terragruntTfBinaryPrefix = prefix + ".0." + terragruntTfBinaryPrefix
 	}
 
 	path, pathOk := d.GetOk(pathPrefix)
+	terragruntTfBinary := d.Get(terragruntTfBinaryPrefix).(string)
+
+	// If this value isn't set, ignore whatever is returned from the response.
```

Comment:
Added a long explanation here. Hopefully it's clear enough.
Basically, if the user hasn't set anything in the schema ignore the response. This is to avoid drifts.